src/libraries/pkg/Microsoft.NETCore.Platforms/runtimeGroups.props

```diff
@@ -81,6 +81,10 @@
       <Versions>1;2</Versions>
     </RuntimeGroup>
 
+    <RuntimeGroup Include="WebAssembly">
```

Comment:
This is a product change, did you mean to include this?  I thought this was still under discussion: https://github.com/dotnet/runtime/issues/33327